pkg/manifest/manifest.go

```diff
@@ -31,6 +38,11 @@ type resourceId struct {
 	Namespace string
 }
 
+type ClusterCapabilities struct {
```

Comment:
We want to drop this and use the API type instead.

Reply:
fixed in https://github.com/openshift/library-go/commit/f529590c5da24451b978dd972aade0a4bc3d40bd